arsen.musheghyan/repo/Lesson_3/exercise_04/exercise_04.py

```diff
@@ -0,0 +1,2 @@
+print ("Exercise N 4")
+print ((512-282)/(47*48+5))
```

Comment:
RE: Your output should be "0.1017" not 0.1017249004865104.

Hint: Use decimal point precision from the input function.